apps/nextjs-website/src/lib/webinars/__tests__/webinarQuestions.test.ts

```diff
@@ -0,0 +1,83 @@
+import { mock } from 'jest-mock-extended';
+import * as E from 'fp-ts/lib/Either';
+import {
+  WebinarEnv,
+  insertWebinarQuestion,
+  listWebinarQuestion,
+} from '../webinarQuestions';
+import { PutItemCommand, QueryCommand } from '@aws-sdk/client-dynamodb';
+import { makeDynamodbItemFromWebinarQuestion } from '../dynamodb/codec';
+
+const aWebinarQuestion = {
+  webinarId: 'aWebinarId',
+  givenName: 'aGivenName',
+  familyName: 'aFamilyName',
+  question: 'aQuestion',
+  createdAt: new Date(),
+  expireAt: new Date('2024-01-22T11:36:31.000Z'),
+};
+const aDynamoDBItem = makeDynamodbItemFromWebinarQuestion({
+  ...aWebinarQuestion,
+});
+
+const makeTestWebinarEnv = () => {
+  const nowDate = new Date(0);
+  const dynamodbClientMock = mock<WebinarEnv['dynamodbClient']>();
```

Comment:
```suggestion
  const dynamoDbClientMock = mock<WebinarEnv['dynamodbClient']>();
```
if you accept previous suggestion, then I'd change this as well